man/busctl.xml

```diff
@@ -406,6 +406,19 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--num-matches=<replaceable>NUMBER</replaceable></option></term>
```

Comment:
i am not a fan of needless abbreviations like this in public APIs. if you want to shorten this, then add a short option "-n" as alias or so, but let's give this an expressive name in the long form. maybe --exit-after-messages= or so?

"matches" is a bit confusing even, as by default we don#t do any matching at all.

Reply:
I'd like the this to be spelled `--limit`. I think the meaning is clear and it is a concept that could be applied to other operations, thus it makes sense to have a common command line option for it.

Reply:
No, I am getting out of the way of bikeshedding - which as you can see above, has already happened plenty. Someone wants A, someone wants B, someone wants C. I am ok with anything - but I do not want to be in the middle and pulled from all sides, sorry.

> quickly after merge

That's 2 weeks...

Reply:
Oh man, this is an API. Please let's get this right. This is not mere bikeshedding, in particular the other issues I pointed out, in particular the idle vs. timeout thing above.

I have no interest in this feature, I am not against it either, but I do care that are APIs are uniform and behave systematically and in an expectable fashion. This feature does not qualify as is.

I posted #34796 for now, as a strawman.

Reply:
I already explained that the behaviour is what I needed here: https://github.com/systemd/systemd/pull/33961#discussion_r1803317209
and again to be clear, if anybody wants another option with different behaviour, no objections here.

Sorry, but whether a command line option is named --exit-after-messages or --num-matches or --limit is pretty much the definition of bikeshedding. And again, if you really want a different name, I have no objection, but please discuss it with the other reviewers who also really wanted a different name, and I will be happy with whichever option wins.

Reply:
The name is my fault. I was probably thinking at the time that this option is really only useful in combination with matching (well, I still think that). And `num` is a quite common abbreviation... But any other name is fine with me. `--limit=`, suggested above by @dnicolodi, sounds nice. `--exit-after-messages=` is okay too.